src/linter/block.go

```diff
@@ -80,6 +83,7 @@ type BlockWalker struct {
 	// shared state between all blocks
 	unusedVars   map[string][]node.Node
 	nonLocalVars map[string]struct{} // static, global and other vars that have complex control flow
+
```

Comment:
This empty line looks excessive. gofmt can't remove it, but it shouldn't be there.